src/components/calculator.js

```diff
@@ -10,28 +10,37 @@ export default function Calculator() {
     setValue(calculate(value, digit));
   }
 
+  const textStyle = {
+    fontSize: '2em',
+    color: 'darkslategray',
+    marginTop: '2em',
+  };
+
   return (
-    <div className="calculator-grid">
-      <Screen value={value} />
-      <Button value="AC" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="+/-" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="%" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="÷" bg="orange" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="7" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="8" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="9" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="x" bg="orange" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="4" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="5" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="6" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="-" bg="orange" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="1" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="2" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="3" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="+" bg="orange" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="0" span="span-two" press={(e) => handleClick(e.target.textContent)} />
-      <Button value="." press={(e) => handleClick(e.target.textContent)} />
-      <Button value="=" bg="orange" press={(e) => handleClick(e.target.textContent)} />
-    </div>
+    <>
+      <b style={textStyle}>Let&apos;s do some math!</b>
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, `general layout should match the given wireframes`. Comparing your calculator page layout to that of the given wireframe, your navigation links are centralised instead of being aligned to the right. 
- [ ] The logo `Maths Magicians` is supposed to be aligned to the left
- [ ] Also the `Let's do some math `is supposed to be to the left of the calculator and not on top of the calculator. See screenshots below:
- Your layout
![image](https://github.com/tamoorsaeed22/Math-Magician/assets/62849872/7b299ef1-709e-415d-9c5d-ea696205b451)
- Wireframe layout 
![image](https://github.com/tamoorsaeed22/Math-Magician/assets/62849872/4cede6c1-8b5e-4a88-bb5f-62de1e3bba29)